meta/main.yml

```diff
@@ -13,6 +13,9 @@ galaxy_info:
       versions:
         - stretch
         - buster
+        # Kali linux isn't an option here, but it is based on Debian
+        # testing
```

Comment:
Can this `# testing` be removed? If it is meant to be there, can it be explained a bit more?

Reply:
That's a continuation of the line above. I pulled that from another role so I didn't think much about the spacing. I can capitalize to "Testing" so it's clearer we're talking about the Debian testing build if you'd like.

Reply:
[Debian testing](https://wiki.debian.org/DebianTesting) is the name for the next-to-be-released Debian version.

Reply:
IMHO, capitalizing "Testing" isn't enough to make it clear what is trying to be said here.  Are we trying to say that Kali is currently based on the current Debian test OS (bullseye)?

Reply:
Now I understand (didn't see @jsf9k's comment until after I posted mine).  How about just adding a link to https://wiki.debian.org/DebianTesting after the word "testing"?

Reply:
Yes, that is what is being said.  Kali is always based on Debian testing, which right now is Bullseye.  Once Bullseye is released, the next Kali releases will be based on Bookworm, and so on.

What about "Debian testing" --> "the latest Debian testing release"?  That sounds awkward to me, so perhaps it can be reworded.

Reply:
How about "Debian Testing" with [this link](https://www.kali.org/docs/policy/kali-linux-relationship-with-debian)?  That is the best I can think of.

Reply:
That works for me 👍

Reply:
Hopefully what I adjusted to in b855eb8 works for everyone.

Reply:
Yes, that looks good - thanks!